docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -22,110 +24,58 @@ command:
 
     openssl rand 512 | base64
 
-Internal SSL/TLS configuration
-------------------------------
+Internal TLS configuration
+--------------------------
 
-SSL/TLS is configured in the ``config.properties`` file.  The SSL/TLS on the
-worker and coordinator nodes are configured using the same set of properties.
-Every node in the cluster must be configured. Nodes that have not been
-configured, or are configured incorrectly, are not able to communicate with
-other nodes in the cluster.
+You can configure the coordinator and all workers to encrypt all communication
+with each other using TLS. Every node in the cluster must be configured. Nodes
+that have not been configured, or are configured incorrectly, are not able to
+communicate with other nodes in the cluster.
 
-To enable SSL/TLS for Trino internal communication, do the following:
+In typical deployments you want to enable :doc:`/security/tls` directly on the
```

Comment:
```
In typical deployments, you will typically want to
:ref:`https-secure-directly <enable TLS directly on the coordinator>`
for fully encrypted access to the client by clients.
```
I worry the below is confusing, since it's not a complete config.

Reply:
Makes sense. Updated